core/jazz_admin-api/index.js

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
don't forget to make the runtime as 6.1(installer)

Reply:
not clear